xml/admin_ceph_cephfs.xml

```diff
@@ -383,102 +383,30 @@ setfattr -n ceph.dir.pin -v 0 a/b # "a/b" is now pinned to rank 0
   </para>
 
   <sect2 xml:id="ceph-cephfs-failover-standby">
-   <title>Configuring Standby Daemons</title>
+   <title>Configuring Standby Replay</title>
    <para>
-    There are several configuration settings that control how a daemon will
-    behave while in standby. You can specify them in the
-    <filename>ceph.conf</filename> on the host where the MDS daemon runs. The
-    daemon loads these settings when it starts, and sends them to the monitor.
+    Each &cephfs; file system may be configured to add standby-replay daemons.
+    These standby daemons follow the active MDS's metadata journal to reduce
+    failover time in the event active MDS becomes unavailable. Each active
```

Comment:
"in the event that an active MDS",  or "in the event that the active MDS"?

Sounds better to me. Not sure of the exact meaning.